{
  try {
    rs.next();
    return false;
  }
 catch (  Exception e) {
    return true;
  }
}
